src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/InstallAppsTask.groovy

```diff
@@ -73,6 +114,18 @@ class InstallAppsTask extends AbstractTask {
         return archiveName;
     }
     
+    private String getInstallAppPackages() {
+        if (project.plugins.hasPlugin("ear")) {
+            project.liberty.installapps.installAppPackages = "project"
+        }
```

Comment:
This is to check if we can ignore the installAppPackages value because we're building an EAR project. Might be a little early to add this in, but it's consistent with the Maven plugin.